src/lib/pci_ahci.c

```diff
@@ -2402,7 +2402,17 @@ pci_ahci_hd_init(struct pci_devinst *pi, char *opts)
 static int
 pci_ahci_atapi_init(struct pci_devinst *pi, char *opts)
 {
-	return (pci_ahci_init(pi, opts, 1));
+	char nopts[MAXPATHLEN + 3];
```

Comment:
I don't know what guarantees MAXPATHLEN gives on the `\0` terminator.  `PATH_MAX` seems to have replaced it. And in hyperkit, it is the latter which is used.